docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+
+* Discuss the project's roadmap with project/product managers, and ask them (and other stakeholders) about immediate documentation needs. For your new writers, ask about topics that could use a little extra attention - an opportunity for them to specialize in something.
+* Encourage new writers to sit in during backlog grooming and project demos. Product prioritization can inform your own priorities.
+* Coordinate with engineering teams by attending their standup and planning meetings. You're working towards a common goal, so it helps to know how everyone's contributions fit together.
+* Try an issue tracking tool (eg JIRA or GitHub Issues). Some feel it helps keep them accountable and prevents them from struggling under the workload. Similarly, for big projects, try mapping them out using a tool, to give your team and managers a high-level glance at everyone's workload.
+* Set up a mandatory flag or field to say if an item needs to be in the release notes. This ensures that important items and fixes won't go undocumented.
+* Audit your docs, archiving or updating content that people aren't reading. Alongside PMs, determine which content is outdated or unnecessary, and then divide them among writers on the team. And if you're struggling to persuade colleagues that an audit is necessary, our next article might just help.
+
+--------------------------
+The costs of outdated docs
+--------------------------
+
+This month's discussions included a brief but fruitful conversation about the tangible costs of outdated, inaccurate, or just plain poorly written documentation. It can be helpful to keep data about the value of good documentation on hand, and our documentarians turned up several reports and studies to consider:
+
+- `The State of Unassisted Support 2014 <http://cdn.swcdn.net/creative/pdf/Whitepapers/SW-WP-The-State-of-Unassisted-Support-2014.pdf>`_, Technology Services Industry Association (TSIA). Includes a graph that compares the cost of self-service support with the costs of other methods.
+- `A Good User's Guide Means Fewer Support Calls and Lower Support Costs <www.jstor.org/stable/43095065>`_ by Cathy J. Spencer and Diana Kilbourn Yates. GE Information Services saved $1 million in support costs per year by providing good user documentation with software products.
+- `Currents, A Seasonal Report on Developer Trends in the Cloud: Open Source Edition <https://www.digitalocean.com/currents/october-2018>`_, Digital Ocean. For open source technologies, great documentation can reduce the barrier to entry for contributors, and it's something companies are looking for when they evaluate open source products.
+
+----------------------------------------------
+Dealing with finding inaccuracies in your docs
```

Comment:
That's ... a lot of gerunds. "When you find inaccuracies in your docs"? Or just "Inaccuracies in your docs"?

Reply:
Gerunds gonna keep on gerunding @Bradamant3

Reply:
fixed!